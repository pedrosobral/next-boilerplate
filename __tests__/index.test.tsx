import { render, screen } from '@testing-library/react';
import Home from '../src/pages';
import '@testing-library/jest-dom';

describe('Home', () => {
  it('render correct background color', () => {
    const { container } = render(<Home />)

    expect(container.firstChild).toHaveStyle({
      'background-color': '#222222'
    })
  })
})