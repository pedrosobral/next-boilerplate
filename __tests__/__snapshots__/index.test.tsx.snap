// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  text-align: center;
  height: 100%;
  width: 100%;
  padding: 4rem;
  background-color: #222222;
  color: white;
}

.emotion-1 {
  width: 15rem;
  margin-bottom: 3rem;
}

.emotion-2 {
  font-size: 2.5rem;
  margin-bottom: 1rem;
}

.emotion-3 {
  font-size: 2rem;
  margin-bottom: 5rem;
}

.emotion-4 {
  width: min(25rem, 100%);
}

<main
  class="emotion-0"
>
  <img
    alt="Ilustração de uma pessoa caminhando da esquerda para a direita com a imagem de um átomo ao fundo. Uma planta no lado direito."
    class="emotion-1"
    src="/img/logo.svg"
  />
  <h1
    class="emotion-2"
  >
    React Avançado - Boilerplate
  </h1>
  <p
    class="emotion-3"
  >
    TypeScript, Emotion, React.js powered by Next.js
  </p>
  <img
    alt="Ilustração de uma programadora sentada de frente a tela de um computador segurando uma xícara na mão esquerda"
    class="emotion-4"
    src="/img/programmer.svg"
  />
</main>
`;
